examples/cli-chat/test/cli_chat_test.clj

```diff
@@ -1,16 +1,14 @@
 (ns cli-chat-test
   (:require
     [cli-chat-fixture :refer [std-system-fixture]]
-    [cli-chat.core]
+    [cli-chat.core :refer [app-cfg]]
     [clj-http.client :as http]
     [clojure.edn :as edn]
     [clojure.string :as str]
     [clojure.test :refer [deftest is use-fixtures]]
     [http.async.client :as a-client]))
 
-(def config {})
-
-(use-fixtures :once (partial std-system-fixture config))
+(use-fixtures :once (partial std-system-fixture app-cfg))
```

Comment:
if we make the IO dependant of the system, it would be good to use 
`(deftest behave-test (with-open [system ,,,,] (make-io! system data) (is ,,,,)))` instead of using this fixture.

Reply:
It's more common to use fixtures. and with fixtures u don't need to couple the test cases into one deftest to not restart the whole application.